cron_scripts/config/config.json

```diff
@@ -0,0 +1,5 @@
+{
+	"MAILSTOREHOST":"10.139.28.238",
+	"MAILSTOREUSER":"admin@zimbra.nic.in",
```

Comment:
Are we unable to get admin user/pass from localconfig?

Reply:
No, we can not get password from localconfig